### PR TITLE
Update OKD version to 4.14.0-0.okd-scos-2024-01-10-151818

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

**Comment by anjannath** [comment]
Unable to start okd with this PR, it fails with the following error:
```
DEBU SSH command results: err: Process exited with status 3, output: ○ dnsmasq.service
     Loaded: masked (Reason: Unit dnsmasq.service is masked.)
    Drop-In: /usr/lib/systemd/system/service.d
             └─10-timeout-abort.conf
     Active: inactive (dead)
DEBU Using root access: Executing systemctl enable dnsmasq.service
DEBU Running SSH command: sudo systemctl enable dnsmasq.service
DEBU SSH command results: err: Process exited with status 1, output:
DEBU Making call to close driver server
DEBU (crc) Calling .Close
DEBU Successfully made call to close driver server
DEBU Making call to close connection to plugin binary
Error running post start: Executing systemctl action failed:  ssh command error:
command : sudo systemctl enable dnsmasq.service
err     : Process exited with status 1
: Failed to enable unit: Unit file /etc/systemd/system/dnsmasq.service is masked.
```

**Comment by praveenkumar** [comment]
> Unable to start okd with this PR, it fails with the following error:
> 
> ```
> DEBU SSH command results: err: Process exited with status 3, output: ○ dnsmasq.service
>      Loaded: masked (Reason: Unit dnsmasq.service is masked.)
>     Drop-In: /usr/lib/systemd/system/service.d
>              └─10-timeout-abort.conf
>      Active: inactive (dead)
> DEBU Using root access: Executing systemctl enable dnsmasq.service
> DEBU Running SSH command: sudo systemctl enable dnsmasq.service
> DEBU SSH command results: err: Process exited with status 1, output:
> DEBU Making call to close driver server
> DEBU (crc) Calling .Close
> DEBU Successfully made call to close driver server
> DEBU Making call to close connection to plugin binary
> Error running post start: Executing systemctl action failed:  ssh command error:
> command : sudo systemctl enable dnsmasq.service
> err     : Process exited with status 1
> : Failed to enable unit: Unit file /etc/systemd/system/dnsmasq.service is masked.
> ```

Thanks for testing it, I think it is masked as part of Fedora CoreOS, will check that and make required change to snc side.

**Comment by praveenkumar** [comment]
/unhold

updated to scos bundle which works without any issue.

**Comment by praveenkumar** [comment]
https://github.com/crc-org/crc/actions/runs/7650561920/job/20846847577?pr=3972 so github action also succeed.

**Review by anjannath — APPROVED** [review]

File: .github/workflows/test-okd-bundle.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          sudo rm -f /mnt/swapfile
+
+      - name: Set the crc config
+        run: |
+          crc config set preset okd
+          crc config set network-mode user
+      - name: Setup the crc
+        run: sudo -su $USER crc setup
+      - name: Start the crc
+        run: sudo -su $USER crc start
```
Comment (anjannath) [inline .github/workflows/test-okd-bundle.yml:57]: we can update this later to have `--log-level debug` so in case of failures we'll have more info

**Pull request merged**